lerna.json

```diff
@@ -2,5 +2,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "3.0.6"
+  "version": "3.0.7"
```

Comment:
I believe there is no need to increment the version no. We usually only do this during a new release. Same for the rest of the `package.json`.

Reply:
@jonahtanjz how do we ensure that the version update happens at the next release? Is it automatic? If not, is it better to update the version number now itself? It might help to detect problems with new versions before the next release.

Reply:
Prof @damithc, based on the project management guide for [doing a release](https://markbind.org/devdocs/devGuide/projectManagement.html#doing-a-release), the version number will be incremented when running the `lerna version --no-push` command. From my understanding, I don't think we should be changing the version number when updating dependencies as this will affect a few things, such as test files etc, and also make it harder to track if for example another PR has incremented the number for another dependency update etc.

>It might help to detect problems with new versions before the next release.

We should still be able to detect issues once the PR is merged to master without changing the version number.

Reply:
Got it. Thanks for the explanation @jonahtanjz